src/main/java/pro/sky/employeeBook/Impl/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,120 @@
+package pro.sky.employeeBook.Impl;
+
+import org.springframework.stereotype.Service;
+import pro.sky.employeeBook.Exception.EmployeeIsInArrayException;
+import pro.sky.employeeBook.Exception.NotFoundEmployeeException;
+import pro.sky.employeeBook.Service.EmployeeService;
+import pro.sky.employeeBook.data.Employee;
+
+import java.util.*;
+import java.util.stream.DoubleStream;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService {
+
+    public Map<String, Employee> employees = new HashMap<>(Map.of(
+            "Тихон Крылов", new Employee("Крылов", "Тихон", 105_853.0, 1),
+            "Маслова Милиса", new Employee("Маслова", "Милиса", 105_853.0, 1),
+            "Яковлев Леонид" ,new Employee("Яковлев", "Леонид", 83_012.0, 2),
+            "Полякова Божена", new Employee("Полякова", "Божена", 119_540.0, 2),
+            "Волков Мирослав" ,new Employee("Волков", "Мирослав", 148_005.0, 3),
+            "Анисимова Данна", new Employee("Анисимова", "Данна", 114_430.0, 3),
+            "Соловьёв Ефрем", new Employee("Соловьёв", "Ефрем", 38_911.0, 4),
+            "Меркушева Глория", new Employee("Меркушева", "Глория", 52_291.0, 4),
+            "Никитин Адам", new Employee("Никитин", "Адам", 66_058.0, 5),
+            "Кудрявцева Галина", new Employee("Кудрявцева", "Галина", 47_832.0, 5)
+    ));
+
+    private String checkingTheString(String string) {
```

Comment:
лучше переименовать в 
```suggestion
    private String checkIsStringAndCapitalize(String string) {
```
поскольку кроме проверки мы делаем и преобразование